kubernetes/src/test/kotlin/com/google/kubernetes/tools/settings/KubernetesSettingsConfigurableTest.kt

```diff
@@ -107,4 +108,12 @@ class KubernetesSettingsConfigurableTest {
 
         Truth.assertThat(kubernetesSettingsConfigurable.isModified).isFalse()
     }
+
+    @Test
+    @UiTest
+    fun `skaffold warning is not shown by default`() {
```

Comment:
Could we also add an opposite test, when Skaffold check fails, the warning is visible?

Reply:
done